From 752aba0d5a67c9cd392f6bcc4f0f15710d875931 Mon Sep 17 00:00:00 2001
From: Roman Stratiienko <r.stratiienko@gmail.com>
Date: Tue, 8 Nov 2022 13:01:37 +0200
Subject: [PATCH] v4l2_request: Add missing includes

... to fix implicit declaration of functions 'close' and 'ioctl' errors

Signed-off-by: Roman Stratiienko <r.stratiienko@gmail.com>
---
 libavcodec/v4l2_request.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libavcodec/v4l2_request.c b/libavcodec/v4l2_request.c
index 824dcaa8e938d7..d550e01c615ebd 100644
--- a/libavcodec/v4l2_request.c
+++ b/libavcodec/v4l2_request.c
@@ -18,10 +18,12 @@
 
 #include <drm_fourcc.h>
 #include <linux/media.h>
+#include <sys/ioctl.h>
 #include <sys/mman.h>
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <fcntl.h>
+#include <unistd.h>
 
 #include <sys/sysmacros.h>
 #include <libudev.h>
-- 
2.37.2

